docs/src/pages/docs/crocks/Identity.md

```diff
@@ -333,12 +333,12 @@ import Identity from 'crocks/Identity'
 
 Identity(42)
   .valueOf()
-//=> 33
+//=> 42
 
-Identity(20)
-  .concat(Identity(22))
+Identity([10,20])
```

Comment:
Wondering why the linter is not picking this up, but gonna wanna add some spaces around the array values like:
`Identity([ 10, 20 ])`